web/themes/joinup/joinup_theme.theme

```diff
@@ -1059,13 +1062,14 @@ function joinup_theme_preprocess_page_title__node__event(&$variables) {
   $field_event_date = $viewBuilder->viewField($node->get('field_event_date'), 'default');
   $field_event_location = $viewBuilder->viewField($node->get('field_location'), 'default');
 
-  $created = \Drupal::service('date.formatter')->format($node->getCreatedTime(), 'date_only');
+  $datetime = empty($node->published_at->value) ? $node->getCreatedTime() : $node->published_at->value;
```

Comment:
Because this is part of the theming. There were no clear instructions for this and this is about the publication date shown in the overview of the content entity itself. Until the initial publication date, only the author can see this. When it becomes public it will get the publication date and that is permanent.
If you want we can have a follow up to change this to also include a "Created at: "/"Published at: " prefix for that time in the theme.

Reply:
Yeah I found it in the meantime as a functional requirement in the initial ticket, this is the intention to show the created date for unpublished content. I would prefer the last updated date, but that is out of scope for this PR.